lib/quantizeAttributes.js

```diff
@@ -0,0 +1,281 @@
+'use strict';
+var Cesium = require('cesium');
+
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var removeUnusedVertices = require('./removeUnusedVertices');
+
+module.exports = quantizeAttributes;
+
+/**
+ * Quantizes attributes in the provided glTF using the WEB3D_quantized_attributes extension.
+ * If options is undefined, all quantizable attributes will be quantized with the default options.
+ * If options.attributes is undefined, all quantizable attributes will be quantized with the provided options.
+ * If options.precision is undefined, the decodeMatrix be written in its full form.
+ *
+ * @param {Object} [gltf] A javascript object holding a glTF hierarchy.
+ * @param {Object} [options=undefined] Defines more specific quantization behavior.
+ * @param {Object} [options.attributes=undefined] Defines specific attributes to be quantized. This should be as arranged as meshId -> primitiveIndex -> [semantic_0, semantic_1, ... semantic_N]
```

Comment:
Do you think it needs to be this precise?  Imagine what a UI for this would look like.

Perhaps just start by providing an array of semantics, e.g., "I want to quantize all POSITIONS."